tsl/src/gapfill/exec.c

```diff
@@ -142,6 +142,8 @@ gapfill_internal_get_datum(int64 value, Oid type)
 static inline int64
 interval_to_usec(Interval *interval)
 {
+	if (interval == NULL)
+		return 0;
```

Comment:
should this ERROR on NULL? it seems like a contract violation to pass in no `interval`

Reply:
changed to error